tools/cpm_tpm_rpk/cpm_tpm_rpk.xml

```diff
@@ -0,0 +1,163 @@
+<tool id="cpm_tpm_rpk" name="Generate CPM, TPM, RPK" version="0.1.0">
+    <description>from raw counts expression values</description>
+    <requirements>
+        <requirement type="package" version="1.6.0=r3.4.2">r-optparse</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Tool exception" />
+    </stdio>
+    <command detect_errors="exit_code"><![CDATA[ 
+Rscript $__tool_directory__/cpm_tpm_rpk.R 
+-d $input 
+-t $option.type_transfo 
+-s $input_sep 
+-c $input_header 
+#if $option.type_transfo == "tpm":
+-g $option.gene_file 
```

Comment:
change the name of this option (-- only option ?) to avoid conflicting optparse option -g